src/envoy/mixer/http_filter.cc

```diff
@@ -129,9 +152,22 @@ class Instance : public Http::StreamFilter, public Http::AccessLog::Instance {
   bool initiating_call_;
   int check_status_code_;
 
+  // mixer control switch (on by default)
+  bool mixer_disabled() {
```

Comment:
the information is available inside the `per route` opaque  config. Not sure if it can be cached on the instance?